src/app/shared/issue/description/description.component.ts

```diff
@@ -104,6 +111,18 @@ export class DescriptionComponent implements OnInit {
     });
   }
 
+  openCancelDialog(): void {
+    const dialogRef = this.dialogService.openUserConfirmationModal(
+      this.cancelEditModalMessages, this.yesButtonModalMessage, this.noButtonModalMessage
+    );
+
+    dialogRef.afterClosed().subscribe(res => {
+      if (res) {
+        this.cancelEditMode();
```

Comment:
Is it possible for us to test this function? i.e. test whether the EditMode has been cancelled if Yes is pressed, not cancelled if No is pressed